common/model/agreement.js

```diff
@@ -75,7 +82,35 @@ class Agreement {
     get signer() {
         return this._signer;
     }
+
+    /**
+     * Returns the signing date.
+     * @returns {Date}
+     */
+    get dateSigned() {
+        return this._dateSigned;
+    }
+
+    save() {
+        const data = {
+            signer: this._signer,
+            type: this._type,
+            dateSigned: this.dateSigned
+        }
+        if (this._organization) {
+            data["organization"] = this._organization
+        }
+
+        console.info("Sending data to FirebaseDB:", data)
+
+        return DB.connection().collection('clas')
```

Comment:
should we call the collection `agreements`?

Reply:
I'm fine with that, I left as it was but it makes sense to call collection with the model name